README.md

```diff
@@ -9,6 +9,18 @@ Exercism exercises in Perl 5
 Each exercise has its own cpanfile (if necessary) for any dependencies it might have.
 Other dependencies can be found in and installed with the top level cpanfile.
 
+It is recommended to use [Carton](https://metacpan.org/pod/Carton).
+Install the Carton module, run `carton install` to install the dependencies from
+the cpanfile, then prepend `carton exec` to your commands to include
+the locally installed dependencies.
+
+## Adding/Updating an Exercise
+
+Each exercise should have an `exercise-data.yaml` file inside a directory named `.meta`
+(please add one if the exercise does not have one yet). This is used by the `exercise-gen.pl`
```

Comment:
This line does not seem to follow the "one sentence per line" as is shown in the lines above this change.

Reply:
I think I would like to see the practice adopted for these files.  One sentence per line, it keeps the diffs cleaner.  Changing one sentence doesn't disrupt the entire paragraph, for example.